working/assets/js/esport.js

```diff
@@ -0,0 +1,191 @@
+(function($){
+	'use strict';
+
+	/*---------- LOADING EFFECT START ----------*/ 
+	setTimeout(function(){
+		$('body').addClass('loaded');
+	}, 2000);
```

Comment:
You need this?
Is not better to put something like "on page load" intead of setTimeout? Because 2s can't be the right value for all users...